### PR TITLE
[WIP] Credit draft implementation.

### PR DESCRIPTION
**Pull request by pgev**

### CODE REVIEW

File: contracts/CreditRule.sol [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+    function executeTransfers(
+        address _from,
+        address[] calldata _transfersTo,
+        uint256[] calldata _transfersAmount
+    )
+        external
+    {
+        require(
+            credits[_from].inProgress,
```
[the remark this answers is not in the document]
Comment (pgev) [reply contracts/CreditRule.sol]: Done.

**Comment by benjaminbollen** [comment]
needs an issue to connect to

**Comment by pgev** [comment]
Closing as a new (updated) PR has been created for Credit (https://github.com/OpenST/openst-contracts/pull/198).

**Comment by pgev** [comment]
Closing as a new (updated) PR has been created for Credit (https://github.com/OpenST/openst-contracts/pull/198).

**Pull request closed without merging**